Configuration/StandardSequences/python/SimL1EmulatorRepack_CalouGT_cff.py

```diff
@@ -11,118 +11,118 @@
 else:
     print("L1T INFO:  L1REPACK:CalouGT (intended for 2016/2017 data), reemulates the Calo part, uses unpacked Muons, and reemulates uGT.")
```

Comment:
Are these printouts still needed (also in other `SimL1EmulatorRepack_*` files)? If no, I'd remove them, if yes, there is likely a (bit hacky) way to recast them as calls to `Modifier.toModify()`.

Reply:
These comments are useful to remind the user of intricacies of the expert level workflow. I'd like to keep them, please.